config/kibana.yml

```diff
@@ -1,8 +1,11 @@
 # Kibana is served by a back end server. This setting specifies the port to use.
 # server.port: 5601
 
-# This setting specifies the IP address of the back end server.
-# server.host: "0.0.0.0"
+# Specifies the host name to which the Kibana server will bind.
+# The default is 'localhost', which usually means remote machines will not be able to connect.
+# To allow connections from remote users, set this parameter to IP or the DNS-name of the machine that is running Kibana.
+# For example: if remote users need to connect to Kibana via http://foobar:5601, set this parameter to 'foobar'.
+# server.host: "localhost"
```

Comment:
Let me take a step back here. What led me down this path is this addition to the config documentation:

```
# To allow connections from remote users, set this parameter to IP or the DNS-name of the machine that is running Kibana.
# For example: if remote users need to connect to Kibana via http://foobar:5601, set this parameter to 'foobar'.
```

I might need to investigate more, but I don't think that doc is correct. As far as I can tell so far, `server.host` does not need to be set to the publicly accessible domain name. It just needs to be set to the publicly accessible IP address of the local machine.

To see this in action:

Add an entry to /etc/hosts (or your OS's equivalent) pointing the hostname `foobar` to 127.0.0.1. Now start up Kibana with `./bin/kibana --host=127.0.0.1`. Go to `foobar:5601` in your browser. Everything works. No need to set server.host to `foobar`.

If you agree with that conclusion, I'd be ok with this change as long as the comment about DNS and foobar is removed.